src/renderer/tvgFrameModule.h

```diff
@@ -39,6 +39,9 @@ class FrameModule: public ImageLoader
     virtual float curFrame() = 0;           //return the current frame number
     virtual float duration() = 0;           //return the animation duration in seconds
 
+    virtual bool getSegment(float* beign, float* end) = 0;      //get the current segment
```

Comment:
The segment feature serves as the base of the animation. It does not need to be a virtual function and should include the segment's body.

Suggestion prototype:
```
void segment(float* beign, float* end);
void segment(float begin, float end);
```